README.md

```diff
@@ -147,49 +147,34 @@ Rev 2 PCBs come with ARM 10-pin SWD headers and can be programmed via ST-Link. A
 ### ST-Link Software Installation
 
 ### Ubuntu
-1.  Download and build the ST-Link utilities:
-    ```
-    git clone https://github.com/stlink-org/stlink
-    cd stlink
-    cmake .
-    make
-    ```
-
-2. Copy the built binaries into their place:
-    ```
-    cd bin
-    sudo cp st-* /usr/local/bin
-    cd ../lib
-    sudo cp *.so* /lib32
-    ```
 
-3. Copy the udev rules to their place (this lets you run the st-link commands without using `sudo`):
-    ```
-    cd ../config/udev/rules.d
-    sudo cp 49-stlinkv* /etc/udev/rules.d/
-    ```
+1. sudo apt-install stlink-tools
```

Comment:
1 is needed for st-flash and st-info things
2 is needed for using the gui
3 is needed for printing. There isnt rly a clean way to see SWO output in ubuntu (even though its trivial on windows). This felt nasty but works nice

Reply:
should i specify what versions ppl need?

Reply:
maybe just the latest verified versions that work